src/manager/components/ThemeBrowser.js

```diff
@@ -1,32 +1,32 @@
 import React from 'react';
 import ReactJson from '@usulpro/react-json-view';
-import { useTheme } from '@storybook/theming';
+import {useTheme} from '@storybook/theming';
```

Comment:
I see the changes like this. It looks like you IDE didn't catch project settings. We should try to keep the existing code style. If your IDE can't solve this - the easiest way is to launch `yarn lint:fix` in the terminal to restore styles

Reply:
Done